docs/modules/plugins/pages/plugin-web-app-to-rest-api.adoc

```diff
@@ -272,3 +272,50 @@ Then I validate HTTP redirects:
 |startUrl                    |endUrl                          |redirectsNumber |
 |http://example.com/redirect |http://example.com/get-response |1               |
 ----
+
+=== Validate SSL rating
+
+:grade: https://github.com/ssllabs/research/wiki/SSL-Server-Rating-Guide[grade]
+
+Performs SSL scanning using https://www.ssllabs.com/index.html[SSL Labs] and compares received {grade} value with expected one.
+
+[source,gherkin]
+----
+Then SSL rating for URL `$url` is $comparisonRule `$gradeName`
+----
+
+* `$url` - The URL for SSL scanning and grading.
+* `$comparisonRule` - xref:parameters:comparison-rule.adoc[The comparison rule].
+* `$gradeName` - The name of {grade}. The possible values: `A+`, `A`, `A-`, `B`, `C`, `D`, `E`, `F`, `T`, `M`.
+
+.Properties
+[cols="3,1,1,3", options="header"]
+|===
+|Property Name
+|Acceptable values
+|Default
+|Description
+
+|`ssl-labs.host`
+|`URL`
+|`https://api.ssllabs.com`
```

Comment:
is ti possible to have private instance of SSL labs? or is it always the same?

Reply:
> is ti possible to have private instance of SSL labs? or is it always the same?

I didn't find information about private instances
But we have SSL Labs DEV Api entry
https://api.dev.ssllabs.com/api/v3/info

Reply:
ok, then it makes sense to keep this property